alerta-covid/spec/requests/api/v1/areas_spec.rb

```diff
@@ -0,0 +1,29 @@
+ require 'rails_helper'
+
+RSpec.describe "/areas", type: :request do
+  let(:valid_attributes) {
```

Comment:
This can be a one-liner

Reply:
Also, I'd add space between brackets when using them 😄 
For example: 
```suggestion
  let(:valid_attributes) { { name: nil } }
```

Reply:
ready, now commits based on these observations